pyanaconda/network.py

```diff
@@ -361,9 +361,13 @@ def networkInitialize(ksdata):
     log.debug("devices found %s", nm.nm_devices())
     logIfcfgFiles("network initialization")
 
-    log.debug("ensure single initramfs connections")
     network_proxy = NETWORK.get_proxy()
-    devnames = network_proxy.ConsolidateInitramfsConnections()
+
+    log.debug("ensure single initramfs connections")
+    task_path = network_proxy.ConsolidateInitramfsConnectionsWithTask()
+    task_proxy = NETWORK.get_proxy(task_path)
+    sync_run_task(task_proxy)
+    devnames = task_proxy.GetResult()
```

Comment:
Please abstract the task creation to a method.

Reply:
This is a correct way how to run a task. I don't know how you want to abstract that.

Reply:
The same code is used on 3 places so it would make sense to have something like:
``
task_path = network_proxy.<something>WithTask()
devnames = self._run_sync_task(task_path)
``

Reply:
@jkonecny12 It seems you werre commenting on already outdated code, something changed in https://github.com/rhinstaller/anaconda/pull/1864/commits/c47f9b35682d7998ed12c441980a533d720b54f7

Reply:
Ahh ok, than you can ignore this. Sorry.